tests/acceptance/DatasetView.feature

```diff
@@ -216,4 +216,19 @@ Feature: a user visit the dataset page
     And I press the button "+"
     And I should see "Alternative names:PYGAD"
     When I press the button "-"
-    Then I should not see "Alternative names:PYGAD"
\ No newline at end of file
+    Then I should not see "Alternative names:PYGAD"
+
+  @ok @issue-2054
+  Scenario: 3D Models tab
+    Given I have not signed in
+    When I am on "/dataset/100006"
+    Then I should see "3D Models"
+
+  @wip @issue-2054
```

Comment:
**suggestion:** Change the `@wip` to `@ok` if the scenario is passing, the wip means `work in progress`.